components/admin/requests/processing/ReviewInformationModal.tsx

```diff
@@ -61,7 +62,10 @@ export default function ReviewInformationModalProps({
                 {requestType === 'REPLACEMENT' ? (
                   <ReasonForReplacementCard applicationId={applicationId} editDisabled />
                 ) : (
-                  <DoctorInformationCard applicationId={applicationId} editDisabled />
+                  <>
+                    <DoctorInformationCard applicationId={applicationId} editDisabled />
+                    <AdditionalInformationCard applicationId={applicationId} />
+                  </>
```

Comment:
For new requests, there will also be a guardian/POA card and a physician assessment card. We will need to add an extra condition block here to account for those sections. We don't have a PhysicianAssessment card yet, but let's add the guardian card for now.